src/Facebook/FacebookClient.php

```diff
@@ -197,7 +197,7 @@ public function prepareRequestMessage(FacebookRequest $request)
      */
     public function sendRequest(FacebookRequest $request)
     {
-        if (get_class($request) === 'FacebookRequest') {
+        if (get_class($request) === 'Facebook\FacebookRequest') {
```

Comment:
It is related since the `get_class()` implementation was broken, but this improvement also makes the code a lot more readable. Do you mind making the change to `instanceof`? :) This should be good to go after that.